pkg/up/up.go

```diff
@@ -535,14 +540,17 @@ func parsePodStatus(pod *v1.Pod) (podStatus, error) {
 	for _, containerStatus := range pod.Status.ContainerStatuses {
 		t := containerStatus.State.Terminated
 		if t != nil {
-			return podStatusOther, fmt.Errorf("aborting because container %s of pod %s terminated (code=%d,signal=%d,reason=%s): %s",
-				containerStatus.Name,
-				pod.ObjectMeta.Name,
-				t.ExitCode,
-				t.Signal,
-				t.Reason,
-				t.Message,
-			)
+			if t.Reason != "Completed" {
+				return podStatusOther, fmt.Errorf("aborting because container %s of pod %s terminated (code=%d,signal=%d,reason=%s): %s",
+					containerStatus.Name,
+					pod.ObjectMeta.Name,
+					t.ExitCode,
+					t.Signal,
+					t.Reason,
+					t.Message,
+				)
+			}
+			return 3, nil
```

Comment:
Can use `podStatusCompleted` instead of `3`

Reply:
Fixed